include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -28,39 +28,49 @@ struct NonUniformBSplinesBase
 
 } // namespace detail
 
-/// NonUniformPointSampling specialization of BSplines
+/**
+ * The type of a non-uniform BSplines 1D basis.
+ *
+ * @tparam Tag The tag identifying the continuous dimension which supports the building of the BSplines.
+ * @tparam D The degree of the BSplines.
+ */
 template <class Tag, std::size_t D>
 class NonUniformBSplines : detail::NonUniformBSplinesBase
 {
     static_assert(D > 0, "Parameter `D` must be positive");
 
 public:
+    /// @brief The tag identifying the continuous dimension which supports the building of the BSplines.
     using tag_type = Tag;
```

Comment:
What is meant by "the continuous dimension which supports the building of the BSplines." ? 

Are you referring to the mathematical term "support" ? I don't know if this can be used as a verb. Even if it can, because "supports" is such a common word I think it is important to mention functions in the same sentence so it is clear that you mean it in this sense
```suggestion
    /// @brief The tag identifying the continuous dimension on which the support of the B-spline functions are defined.
    using tag_type = Tag;
```